manticore/core/plugin.py

```diff
@@ -33,7 +34,8 @@ def context(self):
         return self.manticore.context[plugin_context_name]
 
     def __init__(self):
-        self.manticore = None
+        from manticore import Manticore
```

Comment:
this import is necessary for the type annotation?

Reply:
Yes

Reply:
ok. it strikes me a little odd to put an import here for the sole purpose being able to use the type annotation... ? also, i feel like the type of this attribute is pretty clear, given it's name. so maybe we don't need it?

Reply:
It was just for IDE purposes. I am okay with removing it.